lib/puppet/module.rb

```diff
@@ -67,6 +67,16 @@ def initialize(name, path, environment)
     load_metadata
 
     @absolute_path_to_manifests = Puppet::FileSystem::PathPattern.absolute(manifests)
+
+    # # i18n initialization for modules
+    begin
+      require 'gettext-setup'
+      require 'locale'
+
+      initialize_i18n
+    rescue
+      Puppet.warning "GettextSetup initialization for #{@name} failed."
```

Comment:
So I think since this is going to use Puppet's GettextSetup and not the module's, it can be. And it should be. We are trying to externalize all warnings and errors.